docs/decisions/0004-learner-credit-data-from-analytics-service.rst

```diff
@@ -0,0 +1,46 @@
+4. Learner Credit Usage Data Sourced from Analytics API
+============================================================
+
+Status
+******
+
+Accepted - July 2022
+
+Context
+*******
+
+* Enterprise Admins need a clear and consistent view of how much balance of their Enterprise Offer has been utilized,
+  so they won’t become confused by disagreeing sources.
+
+Decisions
+*********
+
+* We'll rely on the Analytics API to provide enrollment-level Offer utilization data.
+  **This is a subset of the same data source used for the Learner Progress Report.**
+* We'll also rely on the Analytics API to provide aggregate data about total utilization for a given Offer.
+* Both of these will be displayed in the Learner Credit Management page in the admin portal.
+
+Consequences
+************
+
+* We're now bound to the data pipeline for this feature.
+* Fresh data only visible to an end-user after a prefect job transfers data from our data warehouse into
+  the Analytics API.  This was once/day at time of initial implementation, but has been increased to TODO times/day.
+* Pipeline failures mean this data will be stale until failure is resolved.
+* This puts us in a weird position where ecommerce is the source of truth about Enterprise Offer state
+  *except* for the remaining balance for a given Offer.
```

Comment:
Similarly, it's not just about remaining balance, but also the individual offer redemptions (in the table) and the total spend, etc.

Would it be worth saying the inverse here, as well, where the ecommerce data is only currently used for the offer display name and start/end dates?